pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -251,10 +251,19 @@ func (configgen *ConfigGeneratorImpl) buildSidecarOutboundHTTPRouteConfig(env *m
 
 	out := &xdsapi.RouteConfiguration{
 		Name:             routeName,
-		VirtualHosts:     virtualHosts,
 		ValidateClusters: proto.BoolFalse,
 	}
 
+	//BAVERY_TODO: Find a better way to handle these environment variables than parsing each time
+	if enableVHDS, _ := strconv.ParseBool(node.Metadata["ENABLE_DYNAMIC_HOST_CONFIGURATION"]); !enableVHDS {
+		out.VirtualHosts = virtualHosts
```

Comment:
Since unfortunately node metadata is sent on every DiscoveryRequest and Ack - it would be great to use shorter names. They're not user-visible.

Reply:
Changing this to DYNAMIC_CONFIG